cmd/fs-v1.go

```diff
@@ -202,7 +202,7 @@ func (fs *FSObjects) Shutdown(ctx context.Context) error {
 }
 
 // StorageInfo - returns underlying storage statistics.
-func (fs *FSObjects) StorageInfo(ctx context.Context) StorageInfo {
+func (fs *FSObjects) StorageInfo(ctx context.Context, local bool) StorageInfo {
```

Comment:
```suggestion
func (fs *FSObjects) StorageInfo(ctx context.Context, _ bool) StorageInfo {
```

Since we are not using `local` and the name `local` makes less sense in gateways